gateway_test.go

```diff
@@ -129,6 +129,11 @@ func TestMain(m *testing.M) {
 	config.EnableJSVM = true
 	config.Monitor.EnableTriggerMonitors = true
 	config.AnalyticsConfig.NormaliseUrls.Enabled = true
+	config.SlaveOptions.CallTimeout = 30
+	config.SlaveOptions.PingTimeout = 60
+	GlobalRPCPingTimeout = time.Second * time.Duration(config.SlaveOptions.PingTimeout)
+	GlobalRPCCallTimeout = time.Second * time.Duration(config.SlaveOptions.CallTimeout)
```

Comment:
I agree that it's not ideal, but changing how the default config won't work. Note that these settings are applied *after* the config file is read, not before. Perhaps a method like `ensureConfigSanity` to do these last-step tweaks in both prod and test.